hello-world.lke/buildcache/backend.tsbuildinfo

```diff
@@ -0,0 +1,1730 @@
+{
```

Comment:
What is the purpose of this file? I think we can add this into .gitignore  as well.